reV/handlers/transmission.py

```diff
@@ -228,6 +230,22 @@ def _get_features(self, trans_table):
 
         return features
 
+    def _check_feature_dependencies(self):
```

Comment:
Nice! Damn garbage in ;-)

Reply:
Seriously..... The CONUS trans table was solid, no bugs, but i think the CAN table was missing stuff. This helps.